conf/standalone.conf

```diff
@@ -268,3 +268,14 @@ keepAliveIntervalSeconds=30
 
 # How often broker checks for inactive topics to be deleted (topics with no subscriptions and no one connected)
 brokerServicePurgeInactiveFrequencyInSeconds=60
+
+### --- WebSocket --- ###
+
+# Enable the WebSocket API service in broker
+webSocketServiceEnabled=false
```

Comment:
I set it true for now.

I think it's nice to enable to switch WebSocket proxy even when standalone.
I will fix later.
https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/PulsarStandaloneStarter.java#L109